datafusion/core/src/datasource/physical_plan/parquet/statistics.rs

```diff
@@ -75,6 +75,12 @@ macro_rules! get_statistic {
                             *scale,
                         ))
                     }
+                    Some(DataType::Date32) => {
+                        Some(ScalarValue::Date32(Some(*s.$func())))
+                    }
+                    Some(DataType::Date64) => {
```

Comment:
I don't think a Date64 be stored as an `Int32` in parquet. Is this branch required to get the tests to pass? I would be surprised if it were covered

Reply:
Actually, it turns out both Date32 and Date64 seem to be stored in Int32 in parquet.